page-Objects/create-blog-post.pageObject.js

```diff
@@ -0,0 +1,14 @@
+"use strict";
+
+class CreateBlogPost {
+
+	constructor() {
```

Comment:
The code from the actual repo uses two spaces when using TAB, would be nice to maintain the standard.